### ISSUE TITLE
Retrained model produces NaNs

### ISSUE THREAD

**Issue by jjc-sri** [QinjieXiao/EyelashNet#5]
Hi, when I try to retrain the model from scratch, the MSE on the test set converges to NaN after about 2000 iterations.  After some debugging I found that the output of the encoder is roughly 1e13, and this gets amplified by the BasicBlock layers in the decoder until the values are too large (after ~1e36) and get set to NaN.

Is there any advice you can provide?  The large outputs don't make sense, particularly in the context of producing eyelash segmentation masks.

**Comment by QinjieXiao** [comment]
What do you mean by "retraining the model from scratch"? EyelashNet is trained on a pre-trained backbone (https://drive.google.com/open?id=1kNj33D7x7tR-5hXOvxO53QeCEC8ih3-A), not from scratch. Can you show more details about your training set up?

**Comment by jjc-sri** [comment]
Thank you for your reply.  You're right; I am using the pre-trained backbone.  I meant that I'm not resuming from a checkpoint that's already been trained on EyelashNet.

I'm running the training with `python eyelash_train.py --config=config/EyelashNet.toml`

Here is my full config:  [EyelashNet.toml](https://github.com/QinjieXiao/EyelashNet/files/13071865/EyelashNet.txt)
